src/formats/jpeg/frame.zig

```diff
@@ -0,0 +1,247 @@
+const std = @import("std");
+const Allocator = std.mem.Allocator;
+
+const buffered_stream_source = @import("../../buffered_stream_source.zig");
+const Image = @import("../../Image.zig");
+const ImageReadError = Image.ReadError;
+
+const Markers = @import("utils.zig").Markers;
+const FrameHeader = @import("frame_header.zig");
+const QuantizationTable = @import("quantization.zig").Table;
+const HuffmanTable = @import("huffman.zig").Table;
+const color = @import("../../color.zig");
+
+const IDCTMultipliers = @import("utils.zig").IDCTMultipliers;
+const MAX_COMPONENTS = @import("utils.zig").MAX_COMPONENTS;
+const MAX_BLOCKS = @import("utils.zig").MAX_BLOCKS;
+const MCU = @import("utils.zig").MCU;
+
+const Self = @This();
+allocator: Allocator,
```

Comment:
The convention when using struct-as-file like this to name the file PascalCased.zig, thus indicating that the whole file is type unto itself.

Should be renamed to `Frame.zig`